ui/desktop/app/controllers/scopes.js

```diff
@@ -0,0 +1,64 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { loading } from 'ember-loading';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+
+export default class ScopesController extends Controller {
+  // =services
+
+  @service ipc;
+  @service intl;
+  @service flashMessages;
+
+  // =attributes
+
+  metaData;
+  url;
+  @tracked error = false;
+
+  // =methods
+
+  @action
+  @loading
+  async downloadAndInstallCompatiableVersion() {
```

Comment:
```suggestion
  async downloadCompatibleVersion() {
```

We're not really installing it for them

Reply:
Absolutely right. Fixed.